rotkehlchen/tests/api/test_data_import.py

```diff
@@ -428,3 +428,22 @@ def test_data_import_binance_history(rotkehlchen_api_server):
     result = assert_proper_response_with_result(response)
     assert result is True
     assert_binance_import_results(rotki)
+
+
+def test_docker_async_import(rotkehlchen_api_server):
+    """Tests docker async csv import using POST on /import"""
+    dir_path = Path(__file__).resolve().parent.parent
+    filepath = dir_path / 'data' / 'binance_history.csv'
+    response = requests.post(
+        api_url_for(
+            rotkehlchen_api_server,
+            'dataimportresource',
+        ), data={
+            'async_query': True,
+            'source': 'binance',
+        }, files={
+            'file': open(filepath, 'rb'),
+        },
+    )
+    result = assert_proper_response_with_result(response)
+    assert result == {'task_id': 0}
```

Comment:
In my vision we should test here only that import is successfully initialized so we don't need to wait for task completion.

P.S. Test works fine and the Luki`s error had been raising until I've changed the tmp dir behaviour

Reply:
Then change the docstring to signify this. Since you are not testing the actual import here. Describe in the docstring what you are testing.

Reply:
Alright, changed